lib/Mojo/IOLoop.pm

```diff
@@ -230,10 +230,10 @@ Mojo::IOLoop - Minimalistic event loop
 
 =head1 SYNOPSIS
 
-  use Mojo::IOLoop;
+  use Mojo::IOLoop -signatures;
```

Comment:
This is incorrect.

Reply:
Thanks for the review. Would you care to elaborate? Without -signatures, the example does not work for me, as the anonymous subroutines get no arguments.

Reply:
-signatures is a flag for Mojo::Base or Mojolicious::Lite. It is not supported by Mojo::IOLoop. Signatures are assumed in many examples without being specified everywhere; see https://metacpan.org/pod/Mojolicious::Guides#Signatures and [https://metacpan.org/pod/Mojolicious::Guides::FAQ#What-does-%22Illegal-character-in-prototype%22-mean?](https://metacpan.org/pod/Mojolicious::Guides::FAQ#What-does-%22Illegal-character-in-prototype%22-mean?).

Reply:
OK, thanks for explanation. It seems that -signatures is accepted by Mojo::IOLoop nevertheless, as it probably inherits from Mojo::Base.

Removed the -signatures part from the merge request.